c/json.c

```diff
@@ -1047,6 +1102,7 @@ void freeJsonParser(JsonParser *parser) {
 
 static 
 void jsonParseFail(JsonParser *parser, char *formatString, ...) {
+  fflush(stdout);
```

Comment:
What's the purpose of this call? There doesn't seem to be any interaction with stdout here.

Reply:
Dead code according to Joe.

Reply:
fixed in next commit.